documentation/docs/30-advanced/20-hooks.md

```diff
@@ -237,6 +237,36 @@ During development, if an error occurs because of a syntax error in your Svelte
 
 > [!NOTE] Make sure that `handleError` _never_ throws an error
 
+### init
+
+This function is the very first function that will be invoked in the SvelteKit context both on the server (when the server starts) and on the client (when the client side app starts). It can be asynchronous and it will be awaited by the SvelteKit runtime. It's guaranteed to only run once.
+
+On the server it can be used to initialize your database connection, setup your mocks or prepare whatever state your application will need.
+
+> [!NOTE] If your environment supports top level await the `init` function is really not different from writing your initialization in the module itself but it can be useful if you don't have that luxury.
+
+```js
+/// file: src/hooks.server.js
+
+let db;
+
+/** @type {import('@sveltejs/kit').ServerInit} */
+export async function init() {
+	db = await client.connect();
+}
```

Comment:
The example is a bit unrealistic since you wouldn't declare `db` here

```suggestion
import * as db from '$lib/server/database';

/** @type {import('@sveltejs/kit').ServerInit} */
export async function init() {
	await db.connect();
}
```